docs/pages/project/contributing/contributing-ui.md

```diff
@@ -96,6 +96,27 @@ make build-ui
 
 > Changes are not recompiled directly, you will have to run to rebuild the UI to see them
 
+
+#### Build Meshery-UI
```

Comment:
I think we can keep this as a sub heading under `Build and export UI` to make it evident that it a part of building process.